exporter/awsemfexporter/datapoint.go

```diff
@@ -54,7 +56,7 @@ type DataPoint struct {
 type DataPoints interface {
 	Len() int
 	// NOTE: At() is an expensive call as it calculates the metric's value
-	At(i int) DataPoint
+	At(i int) (DataPoint, bool)
```

Comment:
Pls update the comment for the `false` case and explain why do we want to drop those datapoints?

Reply:
Done